examples/jsm/loaders/GLTFLoader.js

```diff
@@ -1814,7 +1814,7 @@ class GLTFTextureTransformExtension {
 
 		if ( transform.texCoord !== undefined ) {
 
-			console.warn( 'THREE.GLTFLoader: Custom UV sets in "' + this.name + '" extension not yet supported.' );
+			texture.uvSet = transform.texCoord;
```

Comment:
@mrdoob this block will need to be moved below the `texture.clone()` line a few lines down, so we don't modify the texture until after cloning it. The next block should check `texture.uvSet !== transform.texCoord` (not than `!== undefined`) so we don't clone unnecessarily.

Reply:
Why do we have to clone?

Reply:
LittlestTokyo.glb is reusing the same texture in multiple materials, with different `texCoord` values. We're overwriting a shared texture, which causes the `uvSet = 1` on a geometry without additional UVs.

Reply:
Ah I see... Hmm, are you able to modify the PR? I think I'm done.

Reply:
Done: https://github.com/mrdoob/three.js/pull/25721/commits/ef9cee5d5d58cafe68728aa6521517ac74d275b6